prusti-common/src/vir/optimizations/functions/inliner.rs

```diff
@@ -76,30 +76,30 @@ fn try_purify(function: &mut ast::Function) -> Option<ast::Expr> {
     None
 }
 
-impl ast::Function {
-    /// Does the function has a body that does not depend neither on
```

Comment:
Why is this code commented out?

Reply:
Since Function and Expr are defined in vir crate now, plus impl for non-current crate things are not possible (unless for a trait that is defined in current crate), so I moved the implementations into vir crate. These were commented out as they are no longer required. I have deleted these lines in the new branch.